index.html

```diff
@@ -6,6 +6,7 @@
     <title>Nep huiswerkopdracht</title>
 </head>
 <body>
```

Comment:
Ik zou hier de lege regel weg laten. Het is ook altijd goed om je code te formatten.